slick.grid.js

```diff
@@ -336,7 +336,7 @@
       $headerScroller = $().add($headerScrollerL).add($headerScrollerR);
 
       // Append the columnn containers to the headers
-      if (options.direction == 'rtl') {//TODO: Changed this
+      if (isRTL()) {//TODO: Changed this
```

Comment:
This TODO should be removed if it is done.
The same may be applied to "the others".